packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -62,6 +66,26 @@ export function TreeHeader(props: TreeHeaderProps): React.ReactElement {
 			>
 				{metadata === undefined ? "" : ` ${metadata}`}
 			</span>
+
+			{tooltipContents === undefined ? (
+				""
+			) : (
+				<Tooltip content={JSON.stringify(tooltipContents)} relationship="description">
+					<span
+						style={{
+							color:
+								themeInfo.name === ThemeOption.HighContrast
+									? ""
```

Comment:
Nit: if this is optional, setting it to `undefined` instead of `""` would probably be better.